maquetado/books-list.html

```diff
@@ -9,6 +9,10 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
+  <div class="books-list-container">
+    <div class="navbar">
+      <img src ="./assets/navbar.png" class="navbar-logo">
```

Comment:
Remember to add the `alt` attribute to your images. This gives them context for screen readers. You could `set alt="Wolox"` in this case